imports/components/@enhancers/recently-liked/index.js

```diff
@@ -0,0 +1,27 @@
+import { graphql } from "react-apollo";
+import gql from "graphql-tag";
+
+const RECENT_LIKES_QUERY = gql`
+  {
+    recentlyLiked(limit: 10, skip: 0) {
+      id
+      ... on Content {
```

Comment:
@edolyne let's make all of the supported like types a fragment that can be shared! Chat with @JakeDawkins about it